collector/container/konflux-slim.Dockerfile

```diff
@@ -74,9 +65,7 @@ RUN mkdir kernel-modules \
     && cp -a /CMakeLists.txt CMakeLists.txt
```

Comment:
GitHub does not let me comment on lines above, and this was in your earlier PR, but I'd still wonder without blocking this PR.

The downstream Dockerfile has
```
ln -s builder/third_party third_party
```

whereas you have
```
&& cp -a /builder/third_party third_party \
```

I suppose both should make the same files visible. Was there a reason you could not use `ln` in Konflux?

Reply:
> Was there a reason you could not use ln in Konflux?

No. 

https://github.com/stackrox/collector/pull/1573/commits/2eadedead357d172fda01842b7c766b6b9839e11